docs/PYPI.md

```diff
@@ -74,61 +79,44 @@ following:
 
 </details>
 
-## Current Features
-
-- Discord Gateway communication
-- logging
-- Http Client
-- Events
-- Event middleware
-- Commands
-- Command arguments *(for types: str, int, float, bool, User, Channel, Role)*
-- Command argument choices
-- Command argument descriptions
-- Command cool downs (Using WindowSliding technique)
-- Tasks
-- Cogs
-
 **Client base class example:**
 
 ```py
 from pincer.client import Bot
 
 # Note that both `Bot` and `Client` are valid!
-bot = Bot("...")
+bot = Bot("YOUR_TOKEN_HERE")
 bot.run()
 ```
 
 **An example on the `on_ready` event**
 
+Pincer makes developing application commands intuitive and fast.
```

Comment:
<details>
<summary>?</summary>
duplicate of L111
</details>
<i>(applies to both files)</i>

Reply:
I changed the first instance to "Pincer bots are required to inherit from the Client."